docs/spec/sanitations.md

```diff
@@ -1,16 +1,18 @@
-_Author_:  <!-- TODO: Add author name --> \
-_Created_: <!-- TODO: Add date --> \
+_Author_:  Thiunuwan \
+_Created_: 11/07/2024 \
 _Updated_: <!-- TODO: Add date --> \
 _Edition_: Swan Lake
 
 # Sanitation for OpenAPI specification
 
 This document records the sanitation done on top of the official OpenAPI specification from Mistral AI. 
-The OpenAPI specification is obtained from (TODO: Add source link).
+The OpenAPI specification is obtained from [Mistral AI OpenAPI specification](https://docs.mistral.ai/redocusaurus/plugin-redoc-0.yaml).
 These changes are done in order to improve the overall usability, and as workarounds for some known language limitations.
 
 [//]: # (TODO: Add sanitation details)
```

Comment:
Lets remove this.

Reply:
Resolved in commit https://github.com/ballerina-platform/module-ballerinax-mistral/pull/3/commits/f4fe9e1b35b6a9c171ab9beeb25ee76fff78614c